src/server/server_family.cc

```diff
@@ -249,6 +249,7 @@ bool IsReplicatingNoOne(string_view host, string_view port) {
   return absl::EqualsIgnoreCase(host, "no") && absl::EqualsIgnoreCase(port, "one");
 }
 
+#ifndef __APPLE__
```

Comment:
to make it less intrusive I would just split `new Transaction{service->FindCmd("FT.CREATE")}};` expression by finding the cid first and returning if it's NULL. Then you do not need these ifdefs. 
i would also add a comment explaining it can be null if dragonfly compiled without search API.

Reply:
sure makes sense will update